scene/gui/line_edit.cpp

```diff
@@ -1315,9 +1315,9 @@ void LineEdit::clear() {
 void LineEdit::show_virtual_keyboard() {
 	if (OS::get_singleton()->has_virtual_keyboard() && virtual_keyboard_enabled) {
 		if (selection.enabled) {
-			OS::get_singleton()->show_virtual_keyboard(text, get_global_rect(), false, max_length, selection.begin, selection.end);
+			OS::get_singleton()->show_virtual_keyboard(text, get_global_rect(), OS::VirtualKeyboardType(virtual_keyboard_type), max_length, selection.begin, selection.end);
```

Comment:
Just a style/consistency suggestion (same below):
```suggestion
			OS::get_singleton()->show_virtual_keyboard(text, get_global_rect(), (OS::VirtualKeyboardType)virtual_keyboard_type, max_length, selection.begin, selection.end);
```